src/rules/fieldsCamelCasedOptionalStartingUnderscore.ts

```diff
@@ -0,0 +1,27 @@
+import { ValidationContext, ASTVisitor } from 'graphql';
+
+import { getNodeName, ValidationError, unwrapAstNode } from '../utils';
+
+const validGraphQLNameTest = RegExp('^[_a-z][a-zA-Z0-9]*$');
```

Comment:
This will allow a field to be named just `_`. Should that be considered valid?

I don't have an opinion, but I wanted to call it out just in case. Maybe it would be good to have another test for single-character names in general.

Reply:
I'll check this but, in theory, this wouldn't be a problem, according to the [GraphQL Spec](https://spec.graphql.org/June2018/#Name).

Reply:
So, I checked, and we decided to keep a single character field name as a valid one, at least for now. It wouldn't be against the GraphQL Spec, at least.